internal/hncconfig/reconciler_test.go

```diff
@@ -301,7 +301,7 @@ var _ = Describe("HNCConfiguration", func() {
 		AddToHNCConfig(ctx, api.RBACGroup, "clusterroles", api.Propagate)
 
 		Eventually(GetHNCConfigCondition(ctx, api.ConditionBadTypeConfiguration, api.ReasonResourceNotNamespaced)).
-			Should(ContainSubstring("Resource \"clusterroles.rbac.authorization.k8s.io\" is not namespaced"))
+			Should(ContainSubstring("resource clusterroles.rbac.authorization.k8s.io is not namespaced"))
```

Comment:
If you use `%q` above we'll keep the double quotes. +1 to correcting the capitalization though, thanks.